web/templates/header.frag

```diff
@@ -0,0 +1,4 @@
+<title>OpenPowerlifting</title>
+<meta charset="utf-8">
+<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="stylesheet" type="text/css" href="style.css"/>
```

Comment:
File needs newline at end. (I think your editor can be configured to do this automatically.)